### PR TITLE
Add CMP compatibility meta tag

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
Add CMP compatibility meta tag. Alternative solution for #157 

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [x] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: Block/Compatibility.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        SearchEngine $searchEngine
+    ) {
+        parent::__construct($context);
+        $this->searchEngine = $searchEngine;
+    }
+
+    /**
+     * Check if Magento uses MySQL as a search engine.
+     *
+     * @return string message
```
Comment (supercid) [inline Block/Compatibility.php:70]:
```suggestion
     * @return bool
```

File: Block/Compatibility.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+     */
+    public function checkSearchEngine()
```
Comment (supercid) [inline Block/Compatibility.php:72]: I'd rename this to `isSearchEngineMysql` or something like that

File: Block/Compatibility.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<?php /** @noinspection PhpUnused */
```
Comment (supercid) [inline Block/Compatibility.php:1]: Which one is unsued? I can't see from the code

Comment (dairbuirabass) [reply Block/Compatibility.php]: Removed

File: Block/Compatibility.php (hunk 2 above)
Comment (dairbuirabass) [reply Block/Compatibility.php]: Renamed 👍

**Review by olsi-qose — APPROVED** [review]

**Pull request merged**